libsel4utils/include/sel4utils/mapping.h

```diff
@@ -67,11 +67,11 @@ void *sel4utils_dup_and_map(vka_t *vka, vspace_t *vspace, seL4_CPtr page, size_t
  */
 void sel4utils_unmap_dup(vka_t *vka, vspace_t *vspace, void *mapping, size_t size_bits);
 
-#if defined(CONFIG_IOMMU) || defined(CONFIG_TK1_SMMU)
+#ifdef defined(CONFIG_IOMMU) || defined(CONFIG_ARM_SMMU) || defined(CONFIG_TK1_SMMU)
```

Comment:
Should this be `#if` instead of `#ifdef`?

```suggestion
#if defined(CONFIG_IOMMU) || defined(CONFIG_ARM_SMMU) || defined(CONFIG_TK1_SMMU)
```